runtimes/ruby-3.1/server.rb

```diff
@@ -3,117 +3,271 @@
 
 USER_CODE_PATH = '/usr/code-start';
 
-before do
-  content_type :json
-end
+class Response
+  def send(body, statusCode = 200, headers = {})
```

Comment:
Variables should be `snake_case` in Ruby, applied to whole PR

Reply:
Updated all